apps/omg_watcher_info/test/omg_watcher_info/pending_block_processor_test.exs

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.PendingBlockProcessorTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: true
+
+  import Ecto.Query, only: [from: 2]
+  import OMG.WatcherInfo.Factory
+
+  alias OMG.WatcherInfo.DB
+  alias OMG.WatcherInfo.DB.Block
+  alias OMG.WatcherInfo.DB.PendingBlock
+  alias OMG.WatcherInfo.PendingBlockProcessor
+
+  @interval 200
+
+  describe "handle_info/2" do
+    setup tags do
+      {:ok, pid} =
+        PendingBlockProcessor.start_link(
+          processing_interval: @interval,
+          name: PendingBlockProcessorTest
+        )
+
+      Map.put(tags, :pid, pid)
+    end
+
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "processes the next pending block in queue", %{pid: pid} do
+      :erlang.trace(pid, true, [:receive])
+
+      assert get_all_pending_blocks() == []
+      assert get_all_blocks() == []
+
+      %{blknum: blknum_1} = insert(:pending_block)
+      %{blknum: blknum_2} = insert(:pending_block)
+      %{blknum: blknum_3} = insert(:pending_block)
+
+      assert_receive {:trace, ^pid, :receive, :timeout}, @interval + 1
```

Comment:
What does `+ 1` represent? I'm assuming it's for the DB insert but not too sure and will be afraid to change. 

Maybe name it in a variable or add as comment to help future maintainers?

Reply:
It's actually not needed, just need to wait for at least `@inteval`, will remove.